src/popup/basics/index.tsx

```diff
@@ -47,6 +49,35 @@ export const Button = ({ size, children, onClick, ...props }: ButtonProps) => (
   </ButtonEl>
 );
 
+/* Back Button */
+interface BackButtonProps {
+  onClick: () => void;
+}
+
+const BackButtonEl = styled.button`
```

Comment:
should this extend `BasicButton` so we get the cursor, focus, etc. styles?

Reply:
Will add it to `ButtonEl` as well! Thanks!